constants/routes.ts

```diff
@@ -13,6 +13,7 @@ export const formatUrl = (route: string, params: Record<string, string>) => {
 export const ROUTES = {
 	Home: {
 		Root: '/',
+		Stats: '/stats',
```

Comment:
I think we can remove this and just route to `ROUTES.Stats.Home` instead? Otherwise it feels like a sub route of home. And unrelated to this PR but same for these dashboard routes under Home, seems they should be moved to under dashboard?

Reply:
Good catch - I can move these around. I originally left stats under "Home" because it could only be access from the homepage, but now that it has it's own route we can lay that out here. The dashboard ones were a similar reason, I think left over from before we had the `/dashboard` route and the complete homepage.